src/pages/Pool/index.tsx

```diff
@@ -165,7 +165,7 @@ function WrongNetworkCard() {
   const theme = useTheme()
 
   return (
-    <>
+    <div style={{ height: '100vh' }}>
       <PageWrapper>
```

Comment:
it looks like `PageWrapper` already has the 100vh style applied--why is the wrapper necessary here?

Reply:
the PageWrapper is not the the outermost div, this extra div is necessary to include SwitchLocaleLink. Putting switchLocaleLink inside PageWrapper has other undesired effects on the Pool component styling